public/search.js

```diff
@@ -51,20 +51,21 @@ $('#dropdown').change(function()
 );
 
 //function to make ajax call to get min and max times
-function ajaxTimesCall(selectedColl)
+var ajaxTimesCall = function(selectedColl)
```

Comment:
Yea no reason; I was trying to get access the function outside of search.js, but I now know this doesn't help at all and that I need to include the js file in the index.html page